packages/phone-number-privacy/signer/src/domain/auth/domainAuth.service.ts

```diff
@@ -1,8 +1,58 @@
+import {
+  DisableDomainRequest,
+  Domain,
+  DomainQuotaStatusRequest,
+  DomainRequest,
+  DomainRestrictedSignatureRequest,
+  SignerEndpoint as Endpoint,
+  verifyDisableDomainRequestAuthenticity,
+  verifyDomainQuotaStatusRequestAuthenticity,
+  verifyDomainRestrictedSignatureRequestAuthenticity,
+} from '@celo/phone-number-privacy-common'
+import Logger from 'bunyan'
+import { DomainStateRecord, DOMAINS_STATES_COLUMNS } from '../../database/models/domainState'
 import { IDomainAuthService } from './domainAuth.interface'
 
+// TODO(Alec): Should we standardize this pattern across signer / combiner?
```

Comment:
Maybe we already discussed this, but I think the current factoring of the combiner works quite well. I don't think we should use this pattern there.

Reply:
Yea I agree, this is a very old TODO statement